azure-pipelines.yml

```diff
@@ -8,8 +8,8 @@ pool:
 
 steps:
 - script: |
-	export APPIUM_BINARY_PATH="/usr/local/lib/node_modules/appium/build/lib/main.js"
-	export NODE_BINARY_PATH="/usr/local/opt/node@8/bin/node"
+    export APPIUM_BINARY_PATH="/usr/local/lib/node_modules/appium/build/lib/main.js"
+    export NODE_BINARY_PATH="/usr/local/opt/node@8/bin/node"
```

Comment:
@ShamaUgale Are you the mac images we use above has node 8 by default?

Reply:
@ShamaUgale you need to revert this as we are not sure if your fix would work, as your overriding user given path of server with APPIUM_BINARY_PATH